crates/storage/src/traits/mod.rs

```diff
@@ -187,6 +187,18 @@ where
 
 /// Pushes the entity to the contract storage using packed layout.
 ///
+/// This is the silent equivalent to the newer version.
+pub fn push_packed_root_compat<T>(entity: &T, root_key: &Key)
```

Comment:
Don't we need the implementation of `Mapping::insert` to use this instead of `push_packed_root`?